source/CargoHold.cpp

```diff
@@ -606,3 +606,23 @@ int CargoHold::IllegalCargoFine() const
 
 	return totalFine;
 }
+
+
+
+// Returns the amount tons of illegal cargo.
+int CargoHold::IllegalCargoAmount() const
+{
+	int count = 0;
+
+	// Find any illegal outfits inside the cargo hold.
+	for(const auto &it : outfits)
+		if(it.first->Get("illegal") || it.first->Get("atrocity") > 0.)
+			count += it.second * max(0., it.first->Mass() + it.first->Get("scan brightness"));
```

Comment:
Uh, am I reading this wrong, or is it getting the Maximum of 0 and a usually-positive number?

Is the intention that `scan brightness` adds to the mass, rather than replacing or scaling it?

Reply:
Yes, that is the intention. The max call also allows scan brightness to be negative to cancel out an outfit's mass, but the total can't go negative.

Reply:
then should that be `min(0., ...)`?

Reply:
It feels to me that having `scan brightness` (if !=0) override mass is a more logical way to do it: since the result when using scan brightness is just an arbitrary number for that outfit, in the present case an arbitrary number based on mass + arbitrary-modifier.

Or (though more convoluted) use the illegal outfit mass as `min(1, actual mass)` then scale that by `scan brightness`, so `2` means "this is twice as bright for its mass" and a massless grenade automatically scans as if it has 1 ton of scannibility.

Reply:
>  then should that be min(0., ...)?

that would always choose the smaller number, so no. 

> It feels to me that having scan brightness (if !=0) override mass is a more logical way to do it:

that would make it impossible to have an stealth outfit that never appears on scans, because we would need to find a way to differentiate between no scan brightness specified and an explicit 0 for stealth.

Reply:
> that would always choose the smaller number, so no.

...I should stop commenting when my head is screwed on the wrong way. Sorry about this! I shall read more carefully before writing any more :-)